src/interaction.js

```diff
@@ -0,0 +1,25 @@
+goog.provide('ngeo.interaction');
```

Comment:
Shouldn't be in `ngeo.interaction.events`?

Reply:
it's not related to events, only common function to all interactions